cpp/tests/search/search_list_test.cpp

```diff
@@ -347,3 +347,66 @@ TYPED_TEST(TypedListContainsTestColumnNeedles, ListsOfStructs)
   auto const result   = cudf::contains(*haystack, *needles);
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected, *result, verbosity);
 }
+
+struct ListLowerBound : public cudf::test::BaseFixture {
+};
+
+TEST_F(ListLowerBound, ListWithNulls)
+{
+  {
+    using lcw = cudf::test::lists_column_wrapper<double>;
```

Comment:
For consistency, please add this line:
```
using doubles_col  = cudf::test::fixed_width_column_wrapper<double>;
```
Into here: https://github.com/rapidsai/cudf/blob/681039ae9e662dd160b91d5db083515357f0f583/cpp/tests/search/search_list_test.cpp#L31